tests/core/admin.py

```diff
@@ -1,6 +1,7 @@
 from django.contrib import admin
 
 from import_export.admin import ExportActionModelAdmin, ImportExportMixin, ImportMixin
+from import_export.fields import Field
```

Comment:
Please can you remove this unused import.

Reply:
removed